<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="jinto" failures="3" tests="5" name="Login test" time="7.553" errors="0" timestamp="2025-03-20T19:20:05 IST">
  <testcase classname="com.hexaclothes.HexaClothesAutomation.BaseTest" name="setup" time="3.292"/>
  <testcase classname="com.hexaclothes.HexaClothesAutomation.LoginTest" name="invalidEmailAndInvalidPassword" time="0.887">
    <failure type="java.lang.AssertionError" message="Test Failed: Login should have failed, but homepage loaded.">
      <![CDATA[java.lang.AssertionError: Test Failed: Login should have failed, but homepage loaded.
at com.hexaclothes.HexaClothesAutomation.LoginTest.invalidEmailAndInvalidPassword(LoginTest.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- invalidEmailAndInvalidPassword -->
  <testcase classname="com.hexaclothes.HexaClothesAutomation.LoginTest" name="invalidEmailAndValidPassword" time="0.597">
    <failure type="java.lang.AssertionError" message="Test Failed: Login should have failed, but homepage loaded.">
      <![CDATA[java.lang.AssertionError: Test Failed: Login should have failed, but homepage loaded.
at com.hexaclothes.HexaClothesAutomation.LoginTest.invalidEmailAndValidPassword(LoginTest.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- invalidEmailAndValidPassword -->
  <testcase classname="com.hexaclothes.HexaClothesAutomation.LoginTest" name="validEmailAndInvalidPassword" time="0.551">
    <failure type="java.lang.AssertionError" message="Test Failed: Login should have failed, but homepage loaded.">
      <![CDATA[java.lang.AssertionError: Test Failed: Login should have failed, but homepage loaded.
at com.hexaclothes.HexaClothesAutomation.LoginTest.validEmailAndInvalidPassword(LoginTest.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- validEmailAndInvalidPassword -->
  <testcase classname="com.hexaclothes.HexaClothesAutomation.LoginTest" name="validUsernameAndValidPassword" time="0.465"/>
</testsuite> <!-- Login test -->
